### ISSUE TITLE
Problem in mIoU

### ISSUE THREAD

**Issue by bravozyz** [HRNet/HRNet-Semantic-Segmentation#228]
Hi~
I'm trying to transform HRNet for semantic segmentation into other deep learning framework, MindSpore, but I got problems in mIoU result. 

**Dataset and Configurations:**
I use Cityscapes as training and evaluation dataset. All settings are same as yours.

**Problems: **
![image](https://user-images.githubusercontent.com/48683684/123364152-7f2ee800-d5a6-11eb-8f3a-60627983113b.png)
This is my result. IoU results of some classes are too low, so that the average, 0.68,  can't achieve the target.

I have tried a lot, such as changing interpolation method in data augmentation from bilinear in nearest, increasing the scaling factor, modifing weights in loss function, changing learning rate. However none of these worked, neither increasing nor decreasing my result.

Are there any other factors contributing to the present results? Can you give me some other suggestions? Thank you very much!

**Comment by PkuRainBow** [comment]
@bravozyz According to your results, I guess the main reason might be failing to load the ImageNet pre-trained weights correctly.
